src/Network/Minio/AdminAPI.hs

```diff
@@ -344,6 +385,35 @@ healPath bucket prefix = do
          <> fromMaybe "" prefix
     else encodeUtf8 $ "v1/heal/"
 
+-- | Get server version and uptime.
+serviceStatus :: Minio ServiceStatus
+serviceStatus = do
+    rsp <- executeAdminRequest AdminReqInfo { ariMethod = HT.methodGet
+                                            , ariPayload = PayloadBS B.empty
+                                            , ariPayloadHash = Nothing
+                                            , ariPath = "v1/service"
+                                            , ariHeaders = []
+                                            , ariQueryParams = []
+                                            }
+
+    let rspBS = NC.responseBody rsp
+    case eitherDecode rspBS of
+        Right ss -> return ss
+        Left err -> throwIO $ MErrVJsonParse $ T.pack err
+
+-- | Send service restart or stop action to Minio server.
+serviceSendAction :: ServiceAction -> Minio ()
+serviceSendAction action = do
+    let payload = PayloadBS $ LBS.toStrict $ A.encode action
+    void $ executeAdminRequest AdminReqInfo { ariMethod = HT.methodPost
+                                            , ariPayload = payload
+                                            , ariPayloadHash = Nothing
+                                            , ariPath = "v1/service"
+                                            , ariHeaders = []
+                                            , ariQueryParams = []
+                                            }
+    return ()
```

Comment:
You can drop the return line because of the void.

Reply:
done